### PR TITLE
Elaborate on CrateDB introduction

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement
First introduction of CrateDB on docs Home page looked like it could be elaborated on a bit. I'm currently going through @simonprickett 's academy materials and have his blessing to borrow bits that could improve the docs.

## Preview
https://cratedb-guide--121.org.readthedocs.build/

## Checklist

 - [ ] Link to issue this PR refers to (if applicable): 
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by matkuliak** [comment]
Just found https://github.com/crate/cratedb-guide/pull/96 
Didn't know we want to make this even shorter. Feel free to close this if that's the case.

File: docs/home/index.md [hunk 1]
```diff
@@ -14,9 +14,14 @@ orphan: true
...
-ease. Here you'll find comprehensive resources including reference manuals,
-tutorials, example code, and more.
+CrateDB is a hyper-fast open source multi-model database combining the
+simplicity of SQL and flexible schemas with the scalability of a distributed
+architecture. CrateDB is a multi-model database. Its strength lies in the
+efficient handling of multiple data models within the same database and even
+within the same record in a table. This allows you to use one database for all
+structured, semi-structured and unstructured data. For example, tabular data, 
+JSON or documents, time series, vectors, geospatial, full text 
+and binary objects
```
Comment (amotl) [inline docs/home/index.md:24]: Can this sentence be improved a bit?

/cc @simonprickett

**Review by amotl — APPROVED** [review]
Thanks! We can also stack up GH-96 later. I think it's always good to have a few fresh words on the topic of explaining CrateDB.

File: docs/home/index.md [hunk 2]
```diff
@@ -14,9 +14,14 @@ orphan: true
 
 # Welcome to CrateDB!
 
-CrateDB is designed to handle your demanding analytics and search workloads with
-ease. Here you'll find comprehensive resources including reference manuals,
-tutorials, example code, and more.
+CrateDB is a hyper-fast open source multi-model database combining the
```
Comment (amotl) [inline docs/home/index.md:17]: Maybe tune that down a bit?
```suggestion
CrateDB is a fast open source multi-model database combining the
```

/cc @seut

File: docs/home/index.md (hunk 1 above)
Comment (matkuliak) [reply docs/home/index.md:24]: Maybe something like:

> This enables you to manage structured, semi-structured, and unstructured data within a single database. CrateDB supports diverse data types such as tabular data, JSON documents, time-series data, vectors, geospatial information, full-text search, and binary objects.

**Review by proddata — CHANGES_REQUESTED** [review]
This change appears to revert to an older, outdated positioning of CrateDB as primarily a multi-model database (including binary objects). Instead, we’ve shifted the focus to CrateDB’s core strengths, particularly leveraging Lucene for fast analytics and search capabilities.

@geragray

**Comment by amotl** [comment]
Thanks.

> Feel free to close this.

So, let's close this and focus on GH-96 instead?

**Comment by matkuliak** [comment]
Sure, that's okay. Thanks!

**Pull request closed without merging**